addon/components/grid-stack.js

```diff
@@ -26,6 +26,7 @@ import { get, action } from '@ember/object';
 import { run } from '@ember/runloop';
 import { capitalize } from '@ember/string';
 import layout from '../templates/components/grid-stack';
+import jQuery from 'jquery';
```

Comment:
That's what I thought, but at the time it was a bit challenging because GridStack was still heavily dependent on jQuery. Now that #56 was merged, it is feasible.